options/locale/locale_en-US.ini

```diff
@@ -681,7 +681,7 @@ choose_new_avatar = Choose new avatar
 update_avatar = Update Avatar
 delete_current_avatar = Delete Current Avatar
 uploaded_avatar_not_a_image = The uploaded file is not an image.
-uploaded_avatar_is_too_big = The uploaded file has exceeded the maximum size.
+uploaded_avatar_is_too_big = The uploaded file size %d KB has exceeded the maximum size %d KB.
```

Comment:
I guess modern days it needs `%.1f MB`

Reply:
> 

For avatars, I think in most case, they should be xx KB or xxx KB but not MB